src/Form/EntityEmbedDialog.php

```diff
@@ -369,10 +368,6 @@ public function buildEmbedStep(array $form, FormStateInterface $form_state) {
       '#type' => 'hidden',
       '#value' => $entity_element['data-entity-type'],
     );
-    $form['attributes']['data-entity-id'] = array(
-      '#type' => 'hidden',
-      '#value' => $entity_element['data-entity-id'],
-    );
     $form['attributes']['data-entity-uuid'] = array(
       '#type' => 'hidden',
       '#value' => $entity_element['data-entity-uuid'],
```

Comment:
And ideally a test that verifies the response that is sent after clicking the submit button.